src/Whoops/Exception/FrameCollection.php

```diff
@@ -181,7 +181,7 @@ public function prependFrames(array $frames)
      * Gets the innermost part of stack trace that is not the same as that of outer exception
      *
      * @param  FrameCollection $parentFrames Outer exception frames to compare tail against
-     * @return Frame[]
+     * @return array
```

Comment:
should not be changed, the more specific one is better

Reply:
What about changing the `$frame` property to `Frame[]` instead of `array`?
https://github.com/filp/whoops/blob/4d363e3711f0dcc6c770b94e24fce80777bc0308/src/Whoops/Exception/FrameCollection.php#L22-L25

Reply:
not sure how this is releated to the change mentioned in this diff, but makes sense.